pkgs/by-name/de/dedoc/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "dedoc";
+  version = "0.2.8";
+
+  src = fetchFromGitHub {
+    owner = "toiletbril";
+    repo = "dedoc";
+    tag = finalAttrs.version;
+    hash = "sha256-edMq/UIHMRHgvML7iv0AhnU+z/4RYlTeKCcV+EhBels=";
+  };
```

It looks like upstream hadn't made a tag for the release.
```suggestion
  version = "0.2.8"; # When upstream releases a new version, update below to tag = ...

  src = fetchFromGitHub {
    owner = "toiletbril";
    repo = "dedoc";
    rev = "1a889de59d642f6b82036b21d0f0f058d8acf9cc";
    hash = "sha256-NCXsinihB/86SgRzER7UfQ6ZbpmyuJowo7fi84PYGfE=";
  };
```